flatpak/jp.yvt.OpenSpades.json

```diff
@@ -0,0 +1,119 @@
+{
+  "app-id": "jp.yvt.OpenSpades",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "18.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "openspades",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--socket=wayland",
+    "--share=network",
+    "--device=dri",
+    "--extension=org.freedesktop.Platform.GL=directory=lib/GL",
+    "--persist=.openspades/"
+  ],
+  "cleanup": [
+    "/include",
+    "/lib/pkgconfig"
+  ],
+  "modules": [
+    "shared-modules/glew/glew.json",
+    {
+      "name": "opus",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://git.xiph.org/opus.git",
+          "tag": "v1.3",
+          "commit": "83d5155f151ca47c9d6274ded1a7481f746b9a43",
+          "disable-shallow-clone": true
+        }
+      ]
+    },
+    {
+      "name": "opusfile",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/xiph/opusfile/releases/download/v0.10/opusfile-0.10.tar.gz",
+          "sha256": "48e03526ba87ef9cf5f1c47b5ebe3aa195bd89b912a57060c36184a6cd19412f"
+        }
+      ]
+    },
+    {
+      "name": "imagemagick",
+      "cleanup": [
+        "/bin",
+        "/etc",
+        "/share",
+        "/lib"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://imagemagick.org/download/ImageMagick.tar.gz",
+          "sha256": "c9e6723aad729140d4af0771d780d20823ca5b0c2fc52cf5a9b2028e923d33d4"
+        }
+      ]
+    },
+    {
+      "name": "openspades",
+      "buildsystem": "cmake",
+      "build-options": {
+        "env": {
+          "CC": "clang",
```

Comment:
I almost forgot this: Is there are reason to use clang instead of gcc?

Reply:
I removed the build options.